data/data/azurestack/bootstrap/main.tf

```diff
@@ -125,7 +119,10 @@ resource "azurestack_virtual_machine" "bootstrap" {
     # isn't installed in RHCOS. As a result, this password is never set. It is
     # included here because it is required by the Azure ARM API.
     admin_password = "NotActuallyApplied!"
-    custom_data    = base64encode(data.ignition_config.redirect.rendered)
+
+    custom_data = base64encode(replace(var.azure_bootstrap_ignition_stub,
+      azurestack_storage_blob.ignition.url,
+    "${azurestack_storage_blob.ignition.url}${data.azurestack_storage_account_sas.ignition.sas}"))
```

Comment:
This is a complicated line so I tried to make it more readable with formatting. I also considered handling this in a local. Not sure if there is any way to make this easier to read.